### PR TITLE
feat: Argo executor config

### PR DESCRIPTION
**Pull request by chrissng**
This PR extends Argo controller's configmap to allow configuring the Argo workflow executor (for e.g. k8s resource usage).

Also, upgraded the image version to v2.6.0

Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by chrissng** [comment]
@benjaminws @seanson able to review this PR?

**Comment by chrissng** [comment]
@benjaminws @seanson able to review this PR?

File: charts/argo/values.yaml [hunk 1]
```diff
@@ -25,6 +25,15 @@ controller:
...
   # podPortName: http
+  executor:
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 0.1
+        memory: 64Mi
+      limits:
+        cpu: 0.5
+        memory: 512Mi
```
Comment (paguos) [inline charts/argo/values.yaml:36]: Can we comment those values? I would not like to force a default.

**Comment by paguos** [comment]
@chrissng I think this was covered in #263. Is it correct?

**Comment by chrissng** [comment]
Thanks for pointing out. Closed this.

**Pull request closed without merging**